.circleci/config.yml

```diff
@@ -0,0 +1,95 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/sigla-UFSCar
+    docker:
+      - image: circleci/ruby:2.4.1-node-browsers
+        environment:
+          PGHOST: 127.0.0.1
```

Comment:
You could add the env `RAILS_ENV` and set it to `test`.

Reply:
Will do it